Client/register.php

```diff
@@ -32,7 +33,7 @@
             if ($isExisted > 0)
             {
                 echo "Email already used. Please use a different email";
-                header("Location: http://localhost/idea-sharing-platform/Client/register.html");
+                header("Location: https://mercury.swin.edu.au/cos10011/s101603196/softwareDev/idea-sharing-platform/Client/register.html");
```

Comment:
I suggest we should use **relative path**  instead of absolute path. This will avoid potential bugs in any of us!

[This article is useful for using relative path for PHP header() function.](https://stackoverflow.com/questions/13366217/php-how-to-header-location-to-a-page-in-a-parent-directory/13366244)